### PR TITLE
Va-minimal-footer: add component

### PR DESCRIPTION
**Pull request by it-harrison**
## Chromatic

https://1777-minimal-footer--60f9b557105290003b387cd5.chromatic.com

---

## Description
Closes [1777](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1777)

## Testing done
local testing in Chrome, Edge, Firefox, Safari

## Screenshots
**Desktop**:

Screenshot 2023-11-06 at 8 48 50 AM

**Mobile**
Simulator Screenshot - iPhone SE (3rd generation) - 2023-11-06 at 08 46 31

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

File: packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx
Comment (micahchiang) [inline packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx:23]: I think because this is a functional image, and serves as the only content of the anchor tag, instead of `aria-hidden="true"` it needs some alt text to indicate that it's a link that takes the user to the va.gov home page. Something like `alt="VA.gov home"` 

More info here - https://www.w3.org/WAI/tutorials/images/functional/#example-1-image-used-alone-as-a-linked-logo. 

And actually, I think the minimal header needs to be fixed as well - https://github.com/department-of-veterans-affairs/component-library/blob/f26ebe31dc5c4adfd445334bf816e8e97ac0e370/packages/web-components/src/components/va-minimal-header/va-minimal-header.tsx#L30

File: packages/web-components/src/index.html
Comment (jamigibbs) [inline packages/web-components/src/index.html:21]: I'm not sure I understand why this file was changed?

File: packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx
Comment (it-harrison) [reply packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx]: @micahchiang thanks for the heads up. Also, [here is a PR](https://github.com/department-of-veterans-affairs/component-library/pull/938) to update minimal header.

File: packages/storybook/public/assets/va-logo-white.png
Comment (jamigibbs) [inline packages/storybook/public/assets/va-logo-white.png:1]: Did we have to manually add this image here to the Storybook assets? Or was it moved by the build script? I noticed that there is a PNG and and SVG version in `web-components/src/assets` but here it's just the PNG.

File: packages/web-components/assets/va-logo-white.png
Comment (jamigibbs) [inline packages/web-components/assets/va-logo-white.png:1]: Same also here. There is a  PNG and and SVG version in `web-components/src/assets` but here it's just the PNG.

File: packages/web-components/src/components/va-minimal-footer/test/va-minimal-footer.spec.tsx
Comment (jamigibbs) [inline packages/web-components/src/components/va-minimal-footer/test/va-minimal-footer.spec.tsx:4]: Do we need a unit test at all for this component? I think we can accomplish the same thing that you're doing here with an e2e test like this:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/web-components/src/components/va-button/test/va-button.e2e.ts#L5-L18

File: packages/storybook/stories/va-minimal-footer.stories.jsx
Comment (jamigibbs) [inline packages/storybook/stories/va-minimal-footer.stories.jsx:17]: The component story isn't display the cod example:

Screenshot 2023-11-07 at 10 43 49 AM

I think to fix this you'll need to pass in an empty object to story like this:

https://github.com/department-of-veterans-affairs/component-library/blob/main/packages/storybook/stories/va-need-help.stories.jsx#L18-L20

File: packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx
Comment (jamigibbs) [reply packages/web-components/src/components/va-minimal-footer/va-minimal-footer.tsx]: I left a [comment on the other PR](https://github.com/department-of-veterans-affairs/component-library/pull/938#discussion_r1385253503) and I just wanted to mention here that we can probably use the existing alt text from the current VA.gov footer logo: "VA logo and Seal, U.S. Department of Veterans Affairs"

There is also a `title` on the link "Go to VA.gov":

Screenshot 2023-11-07 at 2 44 47 PM

File: packages/web-components/src/index.html
Comment (it-harrison) [reply packages/web-components/src/index.html:21]: ug, accidentally committed!

**Comment by jamigibbs** [comment]
@it-harrison  Can you add this new component and the English strings that we will want to get translations for to our spreadsheet here? 🙏 

https://docs.google.com/spreadsheets/d/1iIg9pP5GG-QFk7DrsyoKJv9GebtYmDuzKf_heqxWks4/edit#gid=0

File: packages/storybook/public/assets/va-logo-white.png
Comment (it-harrison) [reply packages/storybook/public/assets/va-logo-white.png:1]: @jamigibbs I removed the .png since it's not being used

File: packages/web-components/assets/va-logo-white.png
Comment (it-harrison) [reply packages/web-components/assets/va-logo-white.png:1]: @jamigibbs I removed the .png since it's not being used

File: packages/web-components/src/components/va-minimal-footer/test/va-minimal-footer.spec.tsx
Comment (it-harrison) [reply packages/web-components/src/components/va-minimal-footer/test/va-minimal-footer.spec.tsx:4]: @jamigibbs I included the unit test here because the similar e2e test required putting the entire base 64 encoded svg in the test which made it unwieldy.

Comment (jamigibbs) [reply packages/web-components/src/components/va-minimal-footer/test/va-minimal-footer.spec.tsx:4]: That makes sense, thank you!

File: packages/storybook/stories/va-minimal-footer.stories.jsx
Comment (jamigibbs) [inline packages/storybook/stories/va-minimal-footer.stories.jsx:19]: You can probably just do 🤷‍♀️:

```
const defaultArgs = {};

const Template = (defaultArgs) => {
```
Or remove `defaultArgs` completely. I think the example I linked to before was done a little strange:

File: packages/web-components/src/index.html
Comment (jamigibbs) [inline packages/web-components/src/index.html:15]: Are you able to revert this change too?

**Review by jamigibbs — APPROVED** [review]
I think dev looks good to me but let's get design approval too before it's merged. cc @danbrady

File: packages/web-components/src/components/va-minimal-footer/va-minimal-footer.scss
Comment (danbrady) [inline packages/web-components/src/components/va-minimal-footer/va-minimal-footer.scss:1]: It doesn't look like we normally have whitespace on the sides of the footer (or header). Can we have the blue background fill the width of the browser regardless of width? (But still have a max-width on a container that gets centered.)

So instead of this currently:

Screenshot 2023-11-09 at 3 11 19 PM

Like this:
Screenshot 2023-11-09 at 3 12 38 PM

Also, looking at the sketch file it looks like we have more padding. Currently it has `padding: 10px 16px`, but Sketch looks more like `padding: 32px 8px` Here's a spec:
 

Screenshot 2023-11-09 at 4 37 39 PM

**Comment by it-harrison** [comment]
@danbrady I have updated the css:
desktop:
Screenshot 2023-11-14 at 10 07 23 AM

I'm not sure if the mobile view is correct - the padding changes from 32px/8px to 10px/16px. I got these numbers from looking at the minimal-header, not sure if they are correct:

Screenshot 2023-11-14 at 10 09 38 AM

**Comment by danbrady** [comment]
Thanks, @it-harrison. Looking at the mockup, we want more height to the footer. So let's set the padding to 32px/8px for both mobile and desktop. When the [header](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1393#issue-1494594540) gets tweaked to fill the width, ee should ensure we're consistent with the padding there too.

**Review by danbrady — APPROVED** [review]

**Pull request merged**
